core/templates/dev/head/pages/exploration_editor/lost_changes_modal.html

```diff
@@ -0,0 +1,22 @@
+<div class="modal-header">
+  <h3>Error With Exploration</h3>
```

Comment:
I've renamed this Error Loading Changes as it will be called when a user attempts to load an exploration but has draft changes stored in local storage which have been overwritten by the saved draft changes. Is that alright?